forestserviceprototype/specialuseform/templates/specialuseform/submitted_permit.html

```diff
@@ -1,10 +1,15 @@
 {% extends "specialuseform/base.html" %}
 
 {% block content %}
-  <h1>Application Submitted</h1>
-  <p class="usa-font-lead">
-    Application #{{ permit.permit_id }}: {{ permit.event_name }}
-  </p>
+  <h1>Recieved Application</h1>
```

Comment:
`Recieved` --> `Received`; if you'd rather I make these kinds of changes directly, lmk.